lib/warnings.js

```diff
@@ -27,4 +27,12 @@ warning.create('FastifyDeprecation', 'FSTDEP013', 'Direct return of "trailers" f
 
 warning.create('FastifyDeprecation', 'FSTDEP014', 'You are trying to set/access the default route. This property is deprecated. Please, use setNotFoundHandler if you want to custom a 404 handler or the wildcard (*) to match all routes.')
 
+warning.create('FastifyDeprecation', 'FSTDEP015', 'The headers schema for the route is undefined. This may indicate the schema is not valid or missing.')
```

Comment:
Please use a single deprecation warning. You can pass a variable inside the message.

```suggestion
warning.create('FastifyDeprecation', 'FSTDEP015', 'The %s schema for the route is undefined. This may indicate the schema is not valid or missing.')
```